README.md

```diff
@@ -76,6 +67,9 @@ metadata:
   name: velero-sample
   namespace: openshift-adp
 spec:
+  features:
+    enableDataMover: true
+    dataMoverCredential: <secret-name>
```

Comment:
Add a note that if this name is not specified then `dm-credentials` is the default name expected by the controller